io/zenoh-transport/tests/unicast_defragmentation.rs

```diff
@@ -93,19 +93,8 @@ async fn run(endpoint: &EndPoint, channel: Channel, msg_size: usize) {
     );
     client_transport.schedule(message.clone()).unwrap();
 
-    // Wait that the client transport has been closed
-    ztimeout!(async {
-        while client_transport.get_zid().is_ok() {
-            tokio::time::sleep(SLEEP).await;
-        }
-    });
-
-    // Wait on the router manager that the transport has been closed
-    ztimeout!(async {
-        while !router_manager.get_transports_unicast().await.is_empty() {
-            tokio::time::sleep(SLEEP).await;
-        }
-    });
+    // wait a little bit for the message to be sent
+    tokio::time::sleep(SLEEP).await;
```

Comment:
Why this change?

Reply:
The test was checkîng that the transport is closed after a defrag error. Since now there is no expected error, I added a sleep to wait for potential errors to happen after the message is sent. I assumed `schedule` does not effectively send the message and that we have to wait for the TX task to pick it up and send it.

Reply:
Then does it make sense at all to keep this test?

Reply:
It seems to be the only fragmentation test we have in the codebase. At least if something goes wrong with fragmentation this test will error out.

Reply:
Other tests trigger fragmentation, like https://github.com/eclipse-zenoh/zenoh/blob/983908b35b2a5bc8830b8eccec8025e62d69dc7f/io/zenoh-transport/tests/unicast_transport.rs#L234

Reply:
Then I guess I'll remove the test since it doesn't serve a purpose anymore.